index.js

```diff
@@ -163,7 +163,8 @@ class ExecutorQueue extends Executor {
      * @return {Promise}
      */
     async postBuildEvent({ pipeline, job, apiUri }) {
-        const jwt = this.tokenGen(Object.keys(pipeline.admins)[0], {}, pipeline.scmContext);
+        const jwt = await this.tokenGen(pipeline.admin
```

Comment:
I don't know well about `async/await` function. Is this `await` working? `tokenGen()` does not return a `Promise`.

Reply:
I'm not sure how that works either. 
But I think this will work:
```
const realAdmin = await pipeline.admin;
const jwt = this.tokenGen(realAdmin.username, {}, pipeline.scmContext);
```

Reply:
I think it does not to `await`. `tokenGen` will start the evaluation after finishing the evaluation of first `Promise` argument.

Reply:
@d2lam This snippet is reasonable for me.
@yoshwata It may just return a mocked value if you check only on test.  I think you should use @d2lam 's snippet.

Reply:
I confirm that a pending `Promise` object is received inside of a function. I add await code like @d2lam suggested.